build.sbt

```diff
@@ -26,8 +26,8 @@ lazy val zioNio = project
   .settings(
     name := "zio-nio",
     libraryDependencies ++= Seq(
-      "dev.zio"        %% "zio"              % "1.0.0-RC12-1",
-      "dev.zio"        %% "zio-streams"      % "1.0.0-RC12-1",
+      "dev.zio"        %% "zio"              % "1.0.0-RC12-1+103-a5c2a8bc",
```

Comment:
is this released yet?

Reply:
No, it's not. It's my local version. I'm waiting for RC14 release where the Executor.asECES is merged.